Source/PLATEAURuntime/Public/CityGML/Serialization/PLATEAUCityObjectSerializationCityModel.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 Ministry of Land, Infrastructure and Transport
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "PLATEAUCityObjectSerializationBase.h"
+
+/**
+* @brief citygml::CityModelをJsonにシリアライズ
+*
+*/
+class PLATEAURUNTIME_API FPLATEAUCityObjectSerializationCityModel : public IPLATEAUCityObjectSerializationBase {
```

Comment:
NativeかUnrealかの違いかと思うので
```suggestion
class PLATEAURUNTIME_API FPLATEAUNativeCityObjectSerialization : public IPLATEAUCityObjectSerializationBase {
```